r2dbc-bom/build.gradle

```diff
@@ -13,3 +13,9 @@ micronautBom {
         acceptedVersionRegressions.add("testcontainers")
     }
 }
+
+micronautBuild {
+    binaryCompatibility {
+        enabled = false
```

Comment:
Build complained because prev version contained reactor-netty-http and now we use mnReactor so disabled check for now. Can enable later